shared/actions/tracker.js

```diff
@@ -83,6 +85,27 @@ export function registerUserChangeListener (): TrackerActionCreator {
   }
 }
 
+export function registerDisplayTLFCreateWithInvite (): TrackerActionCreator {
+  return dispatch => {
+    const params: incomingCallMapType = {
+      'keybase.1.identifyUi.displayTLFCreateWithInvite': (args, response) => {
+        dispatch(({payload: args, type: Constants.showNonUser}: ShowNonUser))
+        response.result()
+      }
+    }
+
+    engine.listenGeneralIncomingRpc(params)
```

Comment:
i was also thinking we should talk about a single place where this kind of stuff happens. like some incoming messages middleware where you can register things. right now its very hard to understand whats registering. don't have to deal with it in this PR but something to talk about